openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/workspace/Tensorflow_Word_Prediction.ipynb

```diff
@@ -113,7 +113,7 @@
    "source": [
     "# First, request a dummy_shard_desc that holds information about the federated dataset \n",
     "dummy_shard_desc = federation.get_dummy_shard_descriptor(size=10)\n",
-    "sample, target = dummy_shard_desc[0]"
+    "sample, target = dummy_shard_desc.get_dataset(dataset_type='train')[0]"
```

Comment:
Maybe better to use `dataset_type=''` because `DummyShardDescriptor.get_dataset` doesn't use this value.

Reply:
yes, you're right, thanks!